### PR TITLE
[staking] switch Ether to native staking V2 at fairbank height

### PR DESCRIPTION
**Pull request by dustinxie**

### CODE REVIEW

File: action/protocol/staking/candidate.go [hunk 1]
```diff
@@ -219,6 +227,14 @@ func (l *CandidateList) Deserialize(buf []byte) error {
 	return nil
 }
 
+func (l CandidateList) toCandidateListV1() (state.CandidateList, error) {
```
Comment (yutongp) [inline action/protocol/staking/candidate.go]: toStateCandidateList

File: action/protocol/poll/staking_committee.go [hunk 2]
```diff
@@ -80,6 +83,7 @@ func NewStakingCommittee(
 		electionCommittee: ec,
 		governanceStaking: gs,
 		nativeStaking:     ns,
+		nativeStaking2:    stk,
```
Comment (yutongp) [inline action/protocol/poll/staking_committee.go]: nativeStakingV2

File: action/protocol/staking/candidate.go (hunk 1 above)
Comment (dustinxie) [reply action/protocol/staking/candidate.go]: update in 2nd commit

File: action/protocol/poll/staking_committee.go (hunk 2 above)
Comment (dustinxie) [reply action/protocol/poll/staking_committee.go]: update in 2nd commit

**Pull request merged**